pydatastructs/trees/heaps.py

```diff
@@ -141,10 +143,10 @@ def insert(self, key, data):
         self.heap.append(new_node)
         self._last_pos_filled += 1
         i = self._last_pos_filled
-        self.heap[i].left, self.heap[i].right = 2*i + 1, 2*i + 2
+        self.heap[i].left, self.heap[i].right = self.d*i + 1, self.d*i + self.d

```

Comment:
May be, `left` -> `leftmost` and `right` -> `rightmost` will sound better.

Reply:
But it may sounds odd for `BinaryHeap`.

Reply:
Two properties can be defined for `BinaryHeap`,
```python
@property
def left(self):
    return self.leftmost

@property
def right(self):
    return self.rightmost
```

Reply:
ok